docs/mux-logging-service.md

```diff
@@ -164,8 +164,8 @@ example, something like this:
     done
 
 If the tag is not in the `project.namespacename` format, the log records will
-use a namespace of `mux-undefined`, so the log records will be in an index
-named something like `project.mux-undefined.$mux-undefined_uuid.YYYY.MM.DD`
+use a namespace of `.orphaned`, so the log records will be in an index
+named something like `project..orphaned.$mux-undefined_uuid.YYYY.MM.DD`
```

Comment:
`.orphaned` isn't a namespace - @jcantrill - what is the full index name for orphaned records?  `.orphaned.orphan-uuid.YYYY.MM.DD`?